init-tools.sh

```diff
@@ -88,33 +88,32 @@ if [ ! -e $__DOTNET_PATH ]; then
                 ;;
 
             Linux)
-            __PKG_RID=linux
+                __PKG_RID=linux
                 OS=Linux
 
                 if [ -e /etc/os-release ]; then
                     source /etc/os-release
                     if [[ $ID == "alpine" ]]; then
                         # remove the last version digit
                         VERSION_ID=${VERSION_ID%.*}
-                        __PKG_RID=alpine.$VERSION_ID
+                    __PKG_RID=alpine.$VERSION_ID
```

Comment:
Indenting `__PKG_RID` earlier looked good, but the rest of the indentation changes look bad.